contrib/trino-connector/src/main/java/io/trino/plugin/starrocks/StarRocksOperationApplier.java

```diff
@@ -0,0 +1,295 @@
+        headers.put("ignore_json_size", "true");
+        this.defaultHeaders = headers.entrySet().stream()
+                .map(entry -> new BasicHeader(entry.getKey(), entry.getValue()))
+                .toArray(Header[]::new);
+    }
+
+    public StreamLoadResponse send()
+    {
+        StreamLoadTableProperties tableProperties = properties.getTableProperties();
+        try {
+            StreamLoadDataFormat dataFormat = tableProperties.getDataFormat();
+            String host = getAvailableHost();
+            String sendUrl = getSendUrl(host, region.getDatabase(), region.getTable());
+            String label = region.getLabel();
+
+            log.info("Stream loading, label : %s, region : %s", label, region.getUniqueKey());
+
+            HttpPut httpPut = new HttpPut(sendUrl);
+            httpPut.setConfig(RequestConfig.custom().setExpectContinueEnabled(true).setRedirectsEnabled(true).build());
+            httpPut.setEntity(new StreamLoadEntity(region, dataFormat, region.getEntityMeta()));
+
+            httpPut.setHeaders(defaultHeaders);
+
+            for (Map.Entry<String, String> entry : tableProperties.getProperties().entrySet()) {
+                httpPut.removeHeaders(entry.getKey());
+                httpPut.addHeader(entry.getKey(), entry.getValue());
+            }
+
+            httpPut.addHeader("label", label);
+
+            try (CloseableHttpClient client = clientBuilder.build()) {
+                log.info("Stream loading, label : %s, request : %s", label, httpPut);
+                long startNanoTime = System.currentTimeMillis();
+                CloseableHttpResponse response = client.execute(httpPut);
+                String responseBody = EntityUtils.toString(response.getEntity());
+
+                log.info("Stream load completed, label : %s, database : %s, table : %s, body : %s",
+                        label, region.getDatabase(), region.getTable(), responseBody);
+
+                StreamLoadResponse streamLoadResponse = new StreamLoadResponse();
+                StreamLoadResponse.StreamLoadResponseBody streamLoadBody = objectMapper.readValue(responseBody, StreamLoadResponse.StreamLoadResponseBody.class);
+                streamLoadResponse.setBody(streamLoadBody);
+
+                String status = streamLoadBody.getStatus();
+
+                if (StreamLoadConstants.RESULT_STATUS_SUCCESS.equals(status)
+                        || StreamLoadConstants.RESULT_STATUS_OK.equals(status)
+                        || StreamLoadConstants.RESULT_STATUS_TRANSACTION_PUBLISH_TIMEOUT.equals(status)) {
+                    streamLoadResponse.setCostNanoTime(System.nanoTime() - startNanoTime);
+                    region.complete(streamLoadResponse);
+                }
+                else if (StreamLoadConstants.RESULT_STATUS_LABEL_EXISTED.equals(status)) {
+                    boolean succeed = checkLabelState(host, region.getDatabase(), label);
+                    if (!succeed) {
```

Comment:
if succeed, should call region.complete

Reply:
done and I would like to add some test cases in the future